cmd/cloud/cluster.go

```diff
@@ -1,17 +1,22 @@
 package main
 
 import (
+	"strings"
+
 	"github.com/spf13/cobra"
 
 	"github.com/mattermost/mattermost-cloud/internal/provisioner"
 )
 
 func init() {
 	clusterCreateCmd.Flags().String("provider", "aws", "Cloud provider hosting the cluster.")
-	clusterCreateCmd.Flags().String("size", "", "The size constant describing the cluster.")
+	clusterCreateCmd.Flags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket used to store cluster state")
+	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster.")
+	clusterCreateCmd.Flags().String("zones", "us-east-1a", "The zones where the cluster will be deployed")
 	clusterCreateCmd.MarkFlagRequired("size")
 
 	clusterDeleteCmd.Flags().String("cluster", "", "The id of the cluster to be deleted.")
+	clusterDeleteCmd.Flags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket where the cluster state is stored")
```

Comment:
So, I actually chose slightly different wording to signify that you needed to provide a specific S3 location where kops could find out about the cluster as opposed to any bucket. I figured it may avoid some confusion in the long run. Thoughts?

Reply:
Ahh ok, I see what you were going for. How about adding "existing" then:

```suggestion
	clusterDeleteCmd.Flags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket where the existing cluster state is stored.")
```

Reply:
I’m not convinced of the need to distinguish the concepts. kops itself [puts](https://github.com/kubernetes/kops/blob/b91db4f360701a7ea90655249ecb7cf68e24545b/cmd/kops/root.go#L130) the flag on the root cmd. It’s perhaps a bit weird if you’re coming to the code without kops context, but I think this is a first class concept for us. Plus, this won’t really scale as we add new cluster commands, e.g. to edit, upgrade.